terrustaria/src/map.rs

```diff
@@ -0,0 +1,182 @@
+use bevy::prelude::*;
+use bevy_ecs_tilemap::prelude::*;
+use bevy_rapier2d::prelude::*;
+use rand::prelude::*;
+
+use crate::constants::*;
+
+struct TileType {
+    #[allow(dead_code)]
+    name: String,
```

Comment:
Używanie stringów zamiast enumów to kiepski pomysł zarówno z punktu widzenia wydajności (stringi zajmują dużo pamięci, enum do 256 wariantów i bez danych mieści się w jednym bajcie), jak i programistycznej wygody: przy wpisaniu niepoprawnego stringa program się wywali, a przy wpisaniu niepoprawnego wariantu program się nie skompiluje.